featuremanagement/_featuremanager.py

```diff
@@ -104,14 +170,16 @@ def is_enabled(self, feature_flag_id, **kwargs):
 
         if not feature_flag.enabled:
             # Feature flags that are disabled are always disabled
-            return False
+            return FeatureManager._check_default_disabled_variant(feature_flag)
 
         feature_conditions = feature_flag.conditions
         feature_filters = feature_conditions.client_filters
 
         if len(feature_filters) == 0:
             # Feature flags without any filters return evaluate
-            return True
+            return FeatureManager._check_default_enabled_variant(feature_flag)
```

Comment:
This should be
```suggestion
            result = True
```

Because variant allocation lower down may override it. And the `_check_default_enabled_variant` function doesn't run the % based allocation.

Reply:
See updated code. Do you think it works now?

Reply:
The updated code should still just be `result["enabled"] = true`, otherwise it will check the default enabled variant, override to false, then later in the code check the disabled default and it could override back to true.